src/reason-parser/reason_syntax_util.ml

```diff
@@ -153,6 +153,17 @@ let escape_string str =
 
 (* #if defined BS_NO_COMPILER_PATCH then *)
 
+(*
+    UTF-8 characters are encoded like this (most editors are UTF-8)
+    0xxxxxxx (length 1)
+    110xxxxx 10xxxxxx (length 2)
+    1110xxxx 10xxxxxx 10xxxxxx (length 3)
+    11110xxx 10xxxxxx 10xxxxxx 10xxxxxx (length 4)
+   Numbers over 127 cannot be encoded in UTF in a single byte, so they use two
+  bytes. That means we can use any characters between 128-255 to encode special
```

Comment:
Great explanation, and thank you for the note. Every time I use/understand one of these printing tricks I had to dig up a Discord conversation.